### DISCUSSION TITLE
Adjust book size to fill screen?

### DISCUSSION THREAD

**Discussion by jGsxrd** [sumatrapdfreader/sumatrapdf#3176] [General]
Using win10. 64-bit v. 3.4.6. 
1/ I use Sumatra PDF as my default EPUB reader. EPUB is by its nature flexible. Yet my pages look like legal size, narrow and tall and the page does not adjust to the full screen width. It's like reading a PDF file which IS a fixed page format. I've tried changing the values in advanced options text to no avail. The page height and width remain the same instead of adjusting to the screen width as it should.

I am NOT talking about full screen. Even in full screen I have 2 wide white margins so book proportions do not adjust ever. See attached picture. If I zoom in to page width, top and bottom of the page disappear. Very annoying. When book height is top to bottom, I want the book to stretch sideways to fill the screen also.

2/ The old advanced settings file allowed to input a font type and size for EPUB books. That seems to have disappeared? While doing internet search, I ran into a picture that mentioned EbookUi with the font type and size I was referring to (2nd picture.) That seems to be completely missing from the current versions and when I put it back in, it keeps disappearing. Is that intentional? Not an improvement imo. This PDF-type fixed 'legal' page size/proportion makes imo the program completely unsuitable as EPUB reader. 

 This is also a matter of **ADA disability accommodation**. I MUST be able to use a larger font across the entire width of the screen without having to zoom out the entire 'fixed' page and losing part of the page that way. So please include that back into newer versions. 

MAY I SUGGEST that — if that is not possible for the PDF reader — you would make a 'sumatra ebook'  version, one that still incorporates the flexibility of an ebook reader and respects the html formatting? Kindle and iOS book readers can do it and still keep text selection and even editing, so it is not impossible.

UPDATE: I have download and installed older versions until I got back what I needed.
I had to go back to v.3.1 before I had the automatic full-screen display AND some of the html formatting back to show paragraph indent. And those older versions have a nice calm background color. That disappeared in later versions too. 
_P.S. It would be helpful if you would include release date of every version on the download page._

Thank you for your time.

![20221217_212454](https://user-images.githubusercontent.com/120875000/208281524-80eec6cb-7b2c-4faa-872d-d6fdae0909e5.jpg)
![ebook toggle](https://user-images.githubusercontent.com/120875000/208282215-48b6ff2c-c52f-41f5-a94d-60ac0be4f475.png)

**Comment by Shiroy0** [comment]
3.1 seems not allow to be downloaded on the official website, so I downloaded 3.2, it's still got ebookUI. 🎉

**Reply by GitHubRulesOK** [reply]
@Shiroy0 
3.12 is a listed address to historic copies so intentionally you need to copy and paste the link

@jGsxrd 
MS Edge is reflowable XMLX (epub) reader for windows, it is slower and has problems but is serviceable.
It does not do read Aloud (perhaps may using narrator/plugin) and does not fill width either
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/e329683c-fcb5-42cb-bdb0-d69c3f0f2638)
but you can zoom
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/0df10038-e7a3-47d2-9400-6e445b62f1fc)

**Comment by zoefoster1** [comment]
I took my ebook formatting services from https://www.barnettghostwriting.com/ they are so professional yet affordable.
